protected/views/site/guide.php

```diff
@@ -318,7 +320,7 @@
                                 </table>
                             </div>
                             <br>
-                            <p>* - Requirements are listed as R= Recommended, O= Optional. Note ^ denotes absolutely mandatory fields.</p>
+                            <p aria-hidden="true">* - Requirements are listed as R= Recommended, O= Optional. Note ^ denotes absolutely mandatory fields.</p>
```

Comment:
this will be ignored by screen readers as in each case the full text is provided